src/test.rs

```diff
@@ -428,3 +428,12 @@ fn test_reentrant() {
     sender.send(null.clone()).unwrap();
     assert_eq!(null, receiver.recv().unwrap());
 }
+
+#[test]
+fn test_large_send() {
+    let payload = vec![5u32; 1024 * 1024 * 12];
```

Comment:
Such a simplistic "pattern" doesn't ensure that the data was actually transferred correctly. IIRC one of the first bugs I fixed in ipc-channel was obscured by that, which is why I modified the existing tests in `src/platform/test.rs`...